lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -14,7 +14,14 @@ class Addon < ::RubyLsp::Addon
 
       sig { returns(RunnerClient) }
       def client
-        @client ||= T.let(RunnerClient.create_client, T.nilable(RunnerClient))
+        @client ||= T.let(
+          if defined?(Rails)
+            RunnerClient.create_client
+          else
+            NullClient.new
```

Comment:
From what I understand `NullClient` is a subclass of `RunnerClient` so there should be no need to do `T.any` nor define an additional. interface

Reply:
Ah yes, I noticed that after I wrote this. Pushed a fix.

I'm still wondering if an interface may be a better safeguard, since if someone does add anything new to `RunnerClient`, they will need to think about waht implementation makes sense for `NullClient`.